#DPATCHLEVEL=0
# Blank lines need to be cleared, else old background shows through
--- src/libxosd/xosd.c~	2004-08-11 18:11:04.000000000 +0200
+++ src/libxosd/xosd.c	2004-08-11 18:10:25.000000000 +0200
@@ -294,14 +294,13 @@ event_loop(void *osdv)
               0, osd->screen_width, osd->line_height);
         }
         switch (osd->lines[line].type) {
-          case LINE_blank:
-            continue;
           case LINE_text:
             draw_text(osd, line);
             break;
           case LINE_percentage:
           case LINE_slider:
             draw_bar(osd, line);
+          case LINE_blank:
             break;
         }
 #ifndef DEBUG_XSHAPE
